testsuite/gnat2goto/tests/type_modular_bitwise/modular_type.adb

```diff
@@ -0,0 +1,22 @@
+procedure Modular_Type is
+  type Less_Than_Ten is mod 10;
+  procedure Assert_Val (X : Less_Than_Ten) is
+     Y : Less_Than_Ten := X;
+     Z : Less_Than_Ten := Y;
+  begin
+     pragma Assert (X = 5);
+     Y := X + 3;
```

Comment:
Why the `+`/`-` here, rather than defining `Y` and `Z` with the right values to begin with?

Reply:
Done.